beestation.dme

```diff
@@ -329,6 +329,7 @@
 #include "code\datums\mind.dm"
 #include "code\datums\mutable_appearance.dm"
 #include "code\datums\mutations.dm"
+#include "code\datums\noobmode.dm"
```

Comment:
Funny, but I reckon it'd be a better idea to give it a more practical filename

Reply:
Any suggestions i dont really know

Reply:
panicbox maybe?